src/components/List/List.stories.tsx

```diff
@@ -11,33 +11,40 @@ export default {
 	component: ListComponent,
 	argTypes: {
 		size: {
-			options: Sizing
+			options: Sizing,
+			control: 'select'
 		},
 		colorTheme: {
-			options: [undefined, ...ColorTheme]
+			options: [undefined, ...ColorTheme],
+			control: 'select'
 		},
 		color: {
 			control: 'radio',
```

Comment:
On the other hand - here we should use `select` because there are too many options 😊